src/services/aws_macros.ts

```diff
@@ -74,6 +76,8 @@ export class AWS {
   route53Client: Route53;
   s3Client: S3;
   snsClient: SNS;
+  stsClient: STS;
```

Comment:
Wait, what is `sts` and why didn't we need to import it in the `package.json`?

Also, why are we not exposing the SQS client in the `aws_sdk` module?

Reply:
Good point. Will expose SQS as well.